try.html

```diff
@@ -1578,6 +1578,13 @@ <h3 id="miscellaneous">Miscellaneous</h3>
           <td><code>https://img.shields.io/github/issues-pr-closed/cdnjs/cdnjs.svg</code></td>
         </tr>
         <tr>
+          <th data-keywords='GitHub pullrequest pr state merge mergeability' data-doc='githubDoc'> GitHub pull request state: </th>
+          <td>
+            <img src='/github/pr-state/badges/shields/6.svg' alt=''/></td>
+          <td>
+            <code>https://img.shields.io/github/pr-state/badges/shields/6.svg</code></td>
+        </tr>
+        <tr>
```

Comment:
After #1163 `try.html` is generated, so could you please make these changes in [lib/all-badge-examples.js](https://github.com/badges/shields/blob/master/lib/all-badge-examples.js) instead, and then run `npm run build`?

`try.html` will no longer be tracked after #1194.